js/client/modules/@arangodb/testutils/test-utils.js

```diff
@@ -166,12 +166,17 @@ function filterTestcaseByOptions (testname, options, whichFilter) {
     whichFilter.filter = 'graph';
     return false;
   }
-  
+
   if (testname.indexOf('-nonwindows') !== -1 && platform.substr(0, 3) === 'win') {
     whichFilter.filter = 'non-windows';
     return false;
   }
 
+  if (testname.indexOf('-nonmac') !== -1 && platform.substr(0, 3) === 'win') {
+    whichFilter.filter = 'non-windows';
```

Comment:
IMHO should rather be named "non-mac"
```suggestion
    whichFilter.filter = 'non-windows';
```